project/npda/templates/partials/visit_form_tab.html

```diff
@@ -0,0 +1,96 @@
+{% load npda_tags %}
+
+<input type="radio"
+  name="visit_tab"
+  role="tab"
+  class="basis-full md:basis-1/3 tab font-bold mb-2 text-black border-b-4 hover:border-b-rcpch_pink border-b-rcpch_light_blue_tint3 border-solid border-0 bg-rcpch_light_blue"
+  aria-label="{{label}} {% if categories|categories_have_errors:visit_instance.errors %} ❗ {% endif %}"
```

Comment:
I don't know how to use the font awesome icon here so I've used an emoji exclamation mark for now